src/main/org/apache/tools/ant/util/jarattr/ClassFormatException.java

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
The commit contains a vim swap file for this class; perhaps .gitignore must be updated?